tsrc/manifest.py

```diff
@@ -34,7 +34,7 @@ def apply_config(self, config: Any) -> None:
         self._handle_groups(groups_config)
 
     def _handle_repo(self, repo_config: Any) -> None:
-        src = repo_config["src"]
+        dest = repo_config["dest"]
```

Comment:
Hi, maybe there could have been a fallback on `src` while parsing the manifest with a warning printed that this is deprecated.

Right now, it's hard to manage the transition to the new format having different tsrc client versions <2.1.0 an >= 2.1.0 on several people computers.

WDYT? Would an MR adding that temporary backward compatibility accepted?